cmd/deploy.go

```diff
@@ -422,25 +422,32 @@ func getChartValues(chartValues map[string]interface{}, clusterName string, depl
 	}
 
 	var overridePaths []string
-
 	useExperimental := viper.GetBool(EXPERIMENTAL_FLAG)
 	if useExperimental {
 		overridePaths = append(overridePaths, EXPERIMENTAL_PRESET_PATH)
 	}
 
-	var resourcesTunerPresetPaths []string
-	if resourcesTunerPresetPaths, err = helm.GetResourcesTunerPresetPaths(deployableNodes); err != nil {
-		return nil, err
+	allocatableResources := helm.CalcAllocatableResources(deployableNodes)
+	sentryHelmContext.AllocatableResources = allocatableResources
```

Comment:
Do we want maybe to store also `deployableNodes`?

Reply:
we store them